docs/getting-started/overview.rst

```diff
@@ -1,6 +1,45 @@
-.. include:: /stub-topic.txt
+Overview of different editions of .NET
+======================================
+
+With the advent of .NET Core and .NET Native, the layout of the .NET ecosystem has changed significantly. Where before there was a (somewhat) simple mapping between platforms and .NET Framework versions, today we have a lot of 
+
+.NET Framework
+--------------
+The tried and true workhorse of the past 10+ years, the full .NET framework is still around, of course, and has been recently updated to version 4.6. There are many enhancements to the CLR (the runtime) and the BCL (the class library) that comes with this release for the consumers presented in the diagram above, such as the .NET Compiler Platform, new language innovations etc.
```

Comment:
I'd steer away from describing .NET Framework in terms of a particular version. Just state what .NET Framework is and provide links for learning more.

You don't need to say that .NET Framework is "not cross-platform". Instead, focus on it being the primary developer experience for most developers on Windows.

Try to avoid terms like "workhorse" and "bulwark", but use simple language that describes the product. I think that this makes us focus on clear descriptions more and is probably helpful for non-English speakers.

> Most of the consumers of this particular edition of the .NET Framework should be already know, if in name only.

Not sure what that means.